client-app/src/components/DraftOrcaDashboard.js

```diff
@@ -76,8 +76,8 @@ const DraftOrcaDashboard = () => {
     axios
       .post("http://localhost:5001/upload", formData)
       .then((response) => {
-        const uploadedFileName = response.data.filename.split("/").pop();
-        setFileName(uploadedFileName);
+        const uploadedFileName = response.data.filename.split(/[/\\]/).pop();
```

Comment:
Please use 'response.data.file_name' to get the filename and assign it to uploadedFileName variable.